plugins/modules/system/sysupgrade.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Andrew Klaus <andrewklaus@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sysupgrade
+
+short_description: Manage OpenBSD system upgrades
+
+version_added: "2.10"
+
+description:
+    - This module can be used to manage OpenBSD system upgrades using sysupgrade.
```

Comment:
```suggestion
    - Manage OpenBSD system upgrades using sysupgrade.
```
If the module can't be used for anything else, we can simplify this, something like ^.

Reply:
My bad, it was like you suggested before, but I told @precurse to change :(